nifi-stateless/nifi-stateless-bundle/nifi-stateless-engine/src/main/java/org/apache/nifi/stateless/engine/StandardStatelessEngine.java

```diff
@@ -733,6 +742,11 @@ public Builder statusTaskInterval(final String statusTaskInterval) {
             return this;
         }
 
+        public Builder statelessEngineConfiguration(final StatelessEngineConfiguration statelessEngineConfiguration) {
+            this.statelessEngineConfiguration = statelessEngineConfiguration;
+            return this;
+        }
```

Comment:
The only thing that we need to pass in here is the timeout - we should avoid passing in the entire StatelessEngineConfiguration and just take in the timeout